portfolio/src/main/webapp/script.js

```diff
@@ -78,3 +77,32 @@ function deleteComment(comment) {
   params.append('id', comment.id);
   fetch('/delete-comment', {method: 'POST', body: params});
 }
+
+function loadMap() {
+  var sanFrancisco = {lat: 37.7749, lng: -122.4194};
```

Comment:
use `const sanFrancisco` instead of `var sanFrancisco`. never use `var` in javascript code because scoping behavior is unexpected.